pkgs/by-name/xo/xonsh/xontribs/xontrib-readable-traceback/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  pkgs,
+  python3,
+}: let
+  backtrace = python3.pkgs.buildPythonPackage rec {
```

Comment:
This should be a separate PR to add backtrace as a direct package into nixpkgs rather than embedding it here.